### PR TITLE
Use xcpretty for xcodebuild on Travis

### PR DESCRIPTION
**Pull request by chinhodado**
Replacement for #842. Continuation of #346 now that xcpretty has supported warnings. [Sample output](https://travis-ci.org/chinhodado/citra/jobs/65013305).

### CODE REVIEW

**Comment by archshift** [comment]
LGTM 420 noscope instamerge.

File: .travis-build.sh [hunk 1]
```diff
@@ -18,5 +18,6 @@ elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
     export Qt5_DIR=$(brew --prefix)/opt/qt5
     mkdir build && cd build
     cmake .. -GXcode
-    xcodebuild -configuration Release
+    gem install xcpretty
```
Comment (archshift) [inline .travis-build.sh]: Why isn't this in `.travis-deps.sh`?

**Comment by archshift** [comment]
LGTM 420 noscope instamerge after that `gem install` is moved to `.travis-deps.sh`

**Comment by chinhodado** [comment]
Done

**Pull request merged**